metrics/default_metrics/tarantool/info.lua

```diff
@@ -0,0 +1,31 @@
+local utils = require('metrics.default_metrics.tarantool.utils');
+
+local function get_info_metrics()
+
+    if not utils.box_is_configured() then
+        return
+    end
+
+    local info = box.info()
+
+
+    utils.set_gauge('info_pid', 'Tarantool pid', info.pid)
+    utils.set_gauge('info_lsn', 'Tarantool lsn', info.lsn)
+    utils.set_gauge('info_uptime', 'Tarantool uptime', info.uptime)
+    utils.set_gauge('info_lsn', 'Tarantool lsn', info.lsn)
+    utils.set_gauge('info_uptime', 'Tarantool uptime', info.uptime)
+
+    for k, v in ipairs(info.vclock) do
+        utils.set_gauge('info_vclock_' .. k, 'VClock for ' .. k, v)
+    end
+
+    for k, v in ipairs(info.replication) do
+        if v.upstream ~= nil then
+            utils.set_gauge('replication_' .. k .. '_lag', 'Replication lag for instance ' .. k, v.upstream.lag)
+        end
+    end
+end
+
+return {
+    init = get_info_metrics,
```

Comment:
Maybe update instead of init?

We usually use init to configure a module for later usage. Think of it as a constructor.

This module doesn't have any state to configure.